src/appleseed.studio/help/about/aboutwindow.cpp

```diff
@@ -88,9 +98,35 @@ void AboutWindow::set_version_string()
             .arg(Appleseed::get_lib_compilation_date())
             .arg(Appleseed::get_lib_compilation_time())
             .arg(Compiler::get_compiler_name())
-            .arg(Compiler::get_compiler_version());
-
-    m_ui->label_version_string->setText(version_string);
+            .arg(Compiler::get_compiler_version()));
+
+    LibraryVersionArray versions = ThirdParties::get_versions();
+    versions.push_back(APIStringPair("Qt", QT_VERSION_STR));
```

Comment:
Why didn't you add Qt's version in the `get_versions()` method ?

Reply:
Because `ThirdParties::get_versions()` is in appleseed.dll which is built without Qt support. Only appleseed.studio is built with Qt support.